Jenkinsfile

```diff
@@ -211,42 +211,39 @@ jobWrapper {
 
             parallel parallelExecutors
         }
-        stage('Aggregate') {
-            parallel (
-                cocoa: {
-                    node('osx') {
-                        getArchive()
-                        for (cocoaStash in cocoaStashes) {
-                            unstash name: cocoaStash
-                        }
-                        sh 'tools/build-cocoa.sh -x'
-                        archiveArtifacts('realm-*-cocoa*.tar.gz')
-                        archiveArtifacts('realm-*-cocoa*.tar.xz')
-                        stash includes: 'realm-*-cocoa*.tar.xz', name: "cocoa-xz"
-                        stash includes: 'realm-*-cocoa*.tar.gz', name: "cocoa-gz"
-                        publishingStashes << "cocoa-xz"
-                        publishingStashes << "cocoa-gz"
-                    }
-                },
-                android: {
-                    node('docker') {
-                        getArchive()
-                        for (androidStash in androidStashes) {
-                            unstash name: androidStash
+        stage('Cocoa xcframeworks') {
+            node('osx') {
+                getArchive()
+                for (cocoaStash in cocoaStashes) {
+                    unstash name: cocoaStash
+                }
+                sh 'tools/build-cocoa.sh -x'
+                archiveArtifacts('realm-*-cocoa*.tar.gz')
+                archiveArtifacts('realm-*-cocoa*.tar.xz')
+                stash includes: 'realm-*-cocoa*.tar.gz', name: "cocoa-gz"
+                stash includes: 'realm-*-cocoa*.tar.xz', name: "cocoa-xz"
+                publishingStashes << "cocoa-gz"
+                publishingStashes << "cocoa-xz"
+            }
+        }
+        stage('Publish packages') {
+            node('docker') {
+                deleteDir()
+                dir('temp') {
+                    withAWS(credentials: 'aws-credentials', region: 'us-east-1') {
```

Comment:
where is `withAWS` defined I don't see it in the shared CI repository

Reply:
It's a step defined by the AWS plugin for Jenkins.